bot/exts/utils/reminders.py

```diff
@@ -209,6 +209,29 @@ async def send_reminder(self, reminder: dict, expected_time: t.Optional[time.Tim
         log.debug(f"Deleting reminder #{reminder['id']} (the user has been reminded).")
         await self.bot.api_client.delete(f"bot/reminders/{reminder['id']}")
 
+    @staticmethod
+    async def try_get_content_from_reply(ctx: Context) -> t.Optional[str]:
+        """
+        Attempts to get content from the referenced message, if applicable.
+
+        Differs from botcore.utils.commands.clean_text_or_reply as allows for embeds.
+        """
+        content = None
+        if reference := ctx.message.reference:
+            if isinstance((resolved_message := reference.resolved), discord.Message):
+                content = resolved_message.content
+
+        # If we weren't able to get the content of a replied message
+        if content is None:
+            await send_denial(ctx, "Your reminder must have a content and/or reply to a message.")
+            return
+
+        # If the replied message has no content (e.g. only attachments/embeds)
+        if content == "":
+            content = "See referenced message."
```

Comment:
Can you put this in italics? to better separate it from a message's content.

Reply:
Yep, done :+1: